source/includes/k8s/deploy-operator.rst

```diff
@@ -65,19 +65,14 @@ The tenant utilizes Persistent Volume Claims to talk to the Persistent Volumes t
 Prerequisites
 -------------
 
-Kubernetes Version 1.21.0
+Kubernetes Version 1.28.0
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. important::
 
    MinIO **strongly recommends** upgrading Production clusters running `End-Of-Life <https://kubernetes.io/releases/patch-releases/#non-active-branch-history>`__ Kubernetes APIs.
 
-Starting with v5.0.0, MinIO **requires** Kubernetes 1.21.0 or later for both the infrastructure and the ``kubectl`` CLI tool.
-
-.. versionadded:: Operator 5.0.6
-
-For Kubernetes 1.25.0 and later, MinIO supports deploying in environments with the :kube-docs:`Pod Security admission (PSA) <concepts/security/pod-security-admission>` ``restricted`` policy enabled.
-
+Starting with v5.0.0, MinIO **requires** Kubernetes 1.28.0 or later for both the infrastructure and the ``kubectl`` CLI tool.
```

Comment:
Which release pushed the minimum required k8s to 1.28? Or was 1.21 incorrect already for 5.0.x?

Reply:
1.21 is EOL and has been for a *while*. I think we just never got around to updating this.

1.28 will be EOL this year, but is at least still in-support (for now).

Reply:
From internal convos it sounds like even if we _do_ technically support older K8s, we want to push users to always stay on latest stable.

I'm going to leave this for now and then see whether there is a way for us to pull this out of the code if we really want to keep a backstop in place.